common/journal_entries/dvg_poland_magnate_oligarchy.txt

```diff
@@ -0,0 +1,145 @@
+﻿je_golden_liberty = {
+    icon = "gfx/interface/icons/event_icons/event_portrait.dds"
+
+    is_shown_when_inactive = {
+        exists = c:POL
+        c:POL = root
+    }
+    
+    should_be_pinned_by_default = yes
+
+    possible = {
+        exists = c:POL
+        c:POL = root
+    }
+
+    complete = {
+        OR = {
+            AND = {
+                ig:ig_landowners = {
+			        is_powerful = yes
+			        is_in_government = yes
+		        }
+		        has_insurrectionary_interest_groups = no
+                owns_entire_state_region = STATE_MAZOVIA
+                has_law = law_type:law_hereditary_bureaucrats
+                has_law = law_type:law_agrarianism
+                has_law = law_type:law_serfdom
+                has_law = law_type:law_oligarchy
+            }
+            AND = {
+                ig:ig_landowners = {
+			        is_powerful = no
+			        is_in_government = no
+		        }
+                has_insurrectionary_interest_groups = no
+                owns_entire_state_region = STATE_MAZOVIA
+                has_law = law_type:law_interventionism
+                has_law = law_type:law_wealth_voting
+                NOT = {
+                    has_law = law_type:law_hereditary_bureaucrats
+                    has_law = law_type:law_serfdom
+                }
+            }
+        }
+    }
+
+    on_complete = {
+        if = {
+            limit = {
+                ig:ig_landowners = {
+			        is_powerful = yes
+			        is_in_government = yes
+		        }
+            }
+            trigger_event = {
+			    id = poland.1
+		    }
+        }
+        else = {
+            trigger_event = {
+			    id = poland.2
+		    }
+        }
+    }
+}
+
+je_uniting_the_commonwealth = {
```

Comment:
![image](https://user-images.githubusercontent.com/116951759/203126388-a6e8d320-e32e-42a3-a165-daae29abc1f3.png)
I think there is something wrong with complete here. I am Poland...
The journal is working though.